File: USSD.pas
Program USSD;
Uses SysUtils;
Var 
	cmd : string;
Begin;
    ReadLn (cmd);
    cmd := StringReplace(cmd, '*', IntToHex(Ord('*')));
    cmd := StringReplace(cmd, '#', IntToHex(Ord('#')));
    WriteLn (cmd);
end.

{ by Tovarisch Trunaev, 2026, Russia, Uray }

File: USSD.pl
#!/usr/bin/perl

use 5.022;
use warnings;

my $cmd = <>;
chomp $cmd;

my $s = sprintf('%X', ord '*');
my $h = sprintf('%X', ord '#');

$cmd =~ s/\*/$s/g;
$cmd =~ s/\#/$h/g;

say $cmd;

1;

# by Tovarisch Trunaev, 2026, Russia, Uray
